src/Turn.js

```diff
@@ -1,29 +1,27 @@
 import domUpdates from './domUpdates';
 
 class Turn {
-  constructor(game, round) {
-    this.game = game;
-    this.round = round;
-    this.currentPlayer = this.game.playerOne;
+  constructor() {
   }
 
-  checkGuess(guess) {
-    let index = this.round.currentSurveyAnswers.findIndex(answerObj => answerObj.answer.toLowerCase() === guess.toLowerCase());
+  checkGuess(game, guess) {
+    let index = game.currentRound.currentSurveyAnswers.findIndex(answerObj => answerObj.answer.toLowerCase() === guess.toLowerCase());
     if (index !== -1) {
```

Comment:
Could we do a `.includes()` here rather than searching for the index?

Reply:
We're checking for the index and then using the index to splice out the correctly guessed answer.